src/events/qaMention.js

```diff
@@ -15,17 +16,16 @@ module.exports = {
         return;
       }
 
-      if (!message.guild) {
-        return;
-      }
-
       const qaRequestChannel = message.guild.channels.cache.find(
         (channel) => channel.name === QA_MENTION.discordQAChannelName
       );
 
       if (!qaRequestChannel) {
         return await message.reply({
-          content: `Sorry, the channel: #${QA_MENTION.discordQAChannelName} for QA requests was not found!`,
+          content: translateLanguage('qaChannelNotFound').replace(
+            '{{channelName}}',
+            QA_MENTION.discordQAChannelName
+          ),
```

Comment:
translateLanguage function has a way of interpolating variables in the translate, just pass it as [second param](https://github.com/TeamNovaSoft/novabot/blob/main/src/languages/index.js#L63C33-L63C39)